src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.html

```diff
@@ -13,10 +13,40 @@ <h2 class="sidebarHeader heading heading3 title" data-test="section-title">${nam
         class="wizardFormInput wizardFormInputNarrow"
         data-test="dao-name-field"
         input-reference.bind="nameReference"
-        label.bind="name + ' Name'"
+        label="Select DAO or Enter a DAO Name"
       >
-        <pinput-text disabled.bind="disabled" value.bind="data.name & validate"
-          view-model.ref="nameReference"></pinput-text>
+        <autocomplete-select
+          placeholder="Select your DAO..."
+          search-text="No DAO found, add '%VAL%' manually"
+          default-avatar="DAO_placeholder.svg"
+          ref="refSelectDAO">
+          <span slot="button-icon-close"><i class="fas fa-times"></i></span>
+          <span slot="button-icon-add"><i class="fas fa-plus-circle"></i></span>
+          <span slot="button-icon-search"><i class="fas fa-search"></i></span>
+        </autocomplete-select>
+        </pform-input>
+        <pform-input if.bind="isLoadingDAO" class="wizardFormInput wizardFormInputNarrow">
```

Comment:
formatting seems off here.
Like at least `</pform-input> needs to be one indentation to the left.

Can you have a look at the whole file please?